branchLayoutImpl/src/main/java/com/virtuslab/branchlayout/impl/readwrite/BranchLayoutFileWriter.java

```diff
@@ -14,22 +14,23 @@
 import com.virtuslab.branchlayout.api.BranchLayoutException;
 import com.virtuslab.branchlayout.api.IBranchLayout;
 import com.virtuslab.branchlayout.api.manager.IBranchLayoutWriter;
+import com.virtuslab.branchlayout.impl.BranchLayout;
+import com.virtuslab.branchlayout.impl.IndentSpec;
 import com.virtuslab.logger.IPrefixedLambdaLogger;
 import com.virtuslab.logger.PrefixedLambdaLoggerFactory;
 
 @RequiredArgsConstructor
 public class BranchLayoutFileWriter implements IBranchLayoutWriter {
   private static final IPrefixedLambdaLogger LOG = PrefixedLambdaLoggerFactory.getLogger("branchLayout");
 
-  private final Path path;
-
-  private final IndentSpec indentSpec;
-
   @Override
   public void write(IBranchLayout branchLayout, boolean backupOldFile) throws BranchLayoutException {
-    LOG.debug(
-        () -> "Entering: branchLayout = ${branchLayout}, backupOldFile = ${backupOldFile}");
-    var lines = printBranchesOntoStringList(branchLayout.getRootEntries(), 0);
+    LOG.debug(() -> "Entering: branchLayout = ${branchLayout}, backupOldFile = ${backupOldFile}");
+
+    var path = ((BranchLayout) branchLayout).getPath();
```

Comment:
This sucks... possibly I should just expose `getIndentSpec()` in `IBranchLayout` instead?... I wanted to keep `IBranchLayout` completely file/indentation/etc.-agnostic, but this might be hard to maintain